lib/ui/page/home/page/chat/view.dart

```diff
@@ -488,6 +488,10 @@ class ChatView extends StatelessWidget {
                               onItemSticky: (i) =>
                                   c.elements.values.elementAt(i)
                                       is DateTimeElement,
+                              forceToExecuteInitIndex:
+                                  c.initIndex == 0 && c.initOffset != 0
+                                  ? 0
+                                  : null,
```

Comment:
Would you be so kind please to write a comment down here for the developers to understand what this thing does here? Time will pass and people will forget.